### ISSUE TITLE
[BUG] Updated to the latest version today, got "Could not load file or assembly" error

### ISSUE THREAD

**Issue by brettofthewild** [MscrmTools/XrmToolBox#1212]
**Describe the bug**
I updated to the latest version of XrmToolbox today using the application; it opened the XrmToolbox autoupdater and ran. Upon trying to re-open, I got the following error:

Could not load file or assembly 'System.Memory, Version=4.0.1.1, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51' or one of its dependencies. The system cannot find the file specified.

**To Reproduce**
Happens each time I start the app.

**Expected behavior**
Well, it not happening.

**Screenshots**
![image](https://user-images.githubusercontent.com/79302547/215849462-d40caa8e-1327-4867-a6d1-924bb7197977.png)

**Desktop (please complete the following information):**
1.2023.1.62

**Additional context**
HALP!

**Comment by brettofthewild** [comment]
Here's the details from the "Error Detail" screen:

**Exception:** System.IO.FileNotFoundException
**Error Code:** blank
**Source:** mscorlib
**Message:** Could not load file or assembly 'System.Memory, Version=4.0.1.1, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51' or one of its dependencies. The system cannot find the file specified.
**Call Stack:**
at System.Signature.GetSignature(Void* pCorSig, Int32 cCorSig, RuntimeFieldHandleInternal fieldHandle, IRuntimeMethodInfo methodHandle, RuntimeType declaringType)
   at System.Reflection.RuntimePropertyInfo.get_Signature()
   at System.RuntimeType.RuntimeTypeCache.MemberInfoCache`1.PopulateProperties(Filter filter, RuntimeType declaringType, Dictionary`2 csPropertyInfos, Boolean[] usedSlots, ListBuilder`1& list)
   at System.RuntimeType.RuntimeTypeCache.MemberInfoCache`1.PopulateProperties(Filter filter)
   at System.RuntimeType.RuntimeTypeCache.MemberInfoCache`1.GetListByName(Char* pName, Int32 cNameLen, Byte* pUtf8Name, Int32 cUtf8Name, MemberListType listType, CacheType cacheType)
   at System.RuntimeType.RuntimeTypeCache.MemberInfoCache`1.Populate(String name, MemberListType listType, CacheType cacheType)
   at System.RuntimeType.GetPropertyCandidates(String name, BindingFlags bindingAttr, Type[] types, Boolean allowPrefixLookup)
   at System.RuntimeType.GetProperties(BindingFlags bindingAttr)
   at System.ComponentModel.Composition.AttributedModel.AttributedPartCreationInfo.<GetExportMembers>d__34.MoveNext()
   at System.Linq.Enumerable.Any[TSource](IEnumerable`1 source)
   at System.ComponentModel.Composition.AttributedModel.AttributedPartCreationInfo.HasExports()
   at System.ComponentModel.Composition.AttributedModel.AttributedPartCreationInfo.IsPartDiscoverable()
   at System.ComponentModel.Composition.AttributedModel.AttributedModelDiscovery.CreatePartDefinitionIfDiscoverable(Type type, ICompositionElement origin)
   at System.ComponentModel.Composition.Hosting.TypeCatalog.get_PartsInternal()
   at System.ComponentModel.Composition.Hosting.TypeCatalog.CreateIndex()
   at System.Lazy`1.CreateValue()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Lazy`1.get_Value()
   at System.ComponentModel.Composition.Hosting.TypeCatalog.GetCandidateParts(String contractName)
   at System.ComponentModel.Composition.Hosting.TypeCatalog.GetCandidateParts(ImportDefinition definition)
   at System.ComponentModel.Composition.Primitives.ComposablePartCatalog.GetExports(ImportDefinition definition)
   at System.Linq.Enumerable.<SelectManyIterator>d__17`2.MoveNext()
   at System.ComponentModel.Composition.Hosting.CatalogExportProvider.InternalGetExportsCore(ImportDefinition definition, AtomicComposition atomicComposition)
   at System.ComponentModel.Composition.Hosting.ExportProvider.TryGetExportsCore(ImportDefinition definition, AtomicComposition atomicComposition, IEnumerable`1& exports)
   at System.ComponentModel.Composition.Hosting.ExportProvider.TryGetExports(ImportDefinition definition, AtomicComposition atomicComposition, IEnumerable`1& exports)
   at System.ComponentModel.Composition.Hosting.CatalogExportProvider.GetExportsCore(ImportDefinition definition, AtomicComposition atomicComposition)
   at System.ComponentModel.Composition.Hosting.ExportProvider.TryGetExportsCore(ImportDefinition definition, AtomicComposition atomicComposition, IEnumerable`1& exports)
   at System.ComponentModel.Composition.Hosting.ExportProvider.GetExports(ImportDefinition definition, AtomicComposition atomicComposition)
   at System.ComponentModel.Composition.Hosting.AggregateExportProvider.GetExportsCore(ImportDefinition definition, AtomicComposition atomicComposition)
   at System.ComponentModel.Composition.Hosting.ExportProvider.TryGetExportsCore(ImportDefinition definition, AtomicComposition atomicComposition, IEnumerable`1& exports)
   at System.ComponentModel.Composition.Hosting.CompositionContainer.GetExportsCore(ImportDefinition definition, AtomicComposition atomicComposition)
   at System.ComponentModel.Composition.Hosting.ExportProvider.TryGetExportsCore(ImportDefinition definition, AtomicComposition atomicComposition, IEnumerable`1& exports)
   at System.ComponentModel.Composition.Hosting.ExportProvider.GetExports(ImportDefinition definition, AtomicComposition atomicComposition)
   at System.ComponentModel.Composition.Hosting.ImportEngine.TryGetExports(ExportProvider provider, ComposablePart part, ImportDefinition definition, AtomicComposition atomicComposition)
   at System.ComponentModel.Composition.Hosting.ImportEngine.TrySatisfyImportSubset(PartManager partManager, IEnumerable`1 imports, AtomicComposition atomicComposition)
   at System.ComponentModel.Composition.Hosting.ImportEngine.TrySatisfyImportsStateMachine(PartManager partManager, ComposablePart part)
   at System.ComponentModel.Composition.Hosting.ImportEngine.TrySatisfyImports(PartManager partManager, ComposablePart part, Boolean shouldTrackImports)
   at System.ComponentModel.Composition.Hosting.ImportEngine.SatisfyImports(ComposablePart part)
   at System.ComponentModel.Composition.Hosting.CompositionServices.TryInvoke(Action action)
   at System.ComponentModel.Composition.Hosting.ComposablePartExportProvider.Compose(CompositionBatch batch)
   at XrmToolBox.PluginManagerExtended.LoadParts(Boolean isRetry)
   at XrmToolBox.PluginManagerExtended.LoadParts(Boolean isRetry)
   at XrmToolBox.PluginManagerExtended.RescanIfRequired()
   at XrmToolBox.PluginManagerExtended.Initialize(Form parentForm)
   at XrmToolBox.New.PluginsForm..ctor()
   at XrmToolBox.New.NewForm..ctor(String[] args)
   at XrmToolBox.Program.Main(String[] args)

**Comment by MscrmTools** [comment]
Thank you for reporting this. 

Looking at it right now

**Comment by MscrmTools** [comment]
@brettofthewild can you open XrmToolBox.exe.config and try to add this node:
```
       <assemblyIdentity name="System.Memory" publicKeyToken="cc7b13ffcd2ddd51" culture="neutral" />
        <bindingRedirect oldVersion="0.0.0.0-4.0.1.2" newVersion="4.0.1.2" />
      </dependentAssembly>
```

And run XrmToolBox

**Comment by MscrmTools** [comment]
So the resolution for this issue is to edit XrmToolBox.exe.config file.
Add the following node to `assemblyBinding` node
```
<assemblyIdentity name="System.Memory" publicKeyToken="cc7b13ffcd2ddd51" culture="neutral" />
        <bindingRedirect oldVersion="0.0.0.0-4.0.1.2" newVersion="4.0.1.2" />
</dependentAssembly>
```

You now should be able to start XrmToolBox.

Please check for update as another issue has been detected and fixed in version 1.2023.1.63
